modules/PM.js

```diff
@@ -0,0 +1,11 @@
+import { Employee } from "./employee.js";
+
+export class PM extends Employee {
+    constructor(id, salary) {
+        super(id, salary);
+    }
+
+    getJobType() {
+        return `I am a PM`
```

Comment:
You need to create a private property for the text "I am a PM" and return it as result of the function getJobType()